### PR TITLE
Angular forms 2023 update

### PR DESCRIPTION
**Pull request by kbritsyn**

### CODE REVIEW

File: angular/modules/forms/README-RU.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # [Angular schedule.](../../README-RU.md) Forms.
 
-## В данном модуле студентам необходимо:
+## ## В данном модуле студентам необходимо изучить:
```
Comment (OlehDulebaEpam) [inline angular/modules/forms/README-RU.md:3]: Is double ## meant to be?

File: tasks/angular/forms.md [hunk 2]
```diff
@@ -41,38 +41,51 @@ The main goal of this task is to implement forms handling and validation using R
...
-- [ ] The application indicated which inputs are invalid on the **Login block** by applying appropriate style and showing corresponding message (**+20**)
+- [ ] Validation rules are applied to the **Login block**. Form submission is possible only if all inputs are valid (**+15**)
+- [ ] The application indicates which inputs are invalid on the **Login block** with appropriate styles and messages (**+15**)
 - [ ] **Admin page** is implemented as reactive form (**+10**)
-- [ ] Validation rules are applied to the **Admin page**. The form submission is possible only if a user enters valid data to all inputs (**+20**)
-- [ ] The application indicated which inputs are invalid on the **Admin page** by applying appropriate style and showing corresponding message (**+20**)
+- [ ] Validation rules are applied to the **Admin page**. Form submission is possible only if all inputs are valid (**+15**)
+- [ ] The application indicates which inputs are invalid on the **Admin page** with appropriate styles and messages (**+15**)
+- [ ] "Tags" sub-form is implemented using `FormArray`. "Add tag" button works correctly (**15**)
```
Comment (OlehDulebaEpam) [inline tasks/angular/forms.md:70]: + sign is missing in (**+15**)

File: tasks/angular/forms.md [hunk 3]
```diff
@@ -41,38 +41,51 @@ The main goal of this task is to implement forms handling and validation using R
...
+- [ ] "Reset" button functionality is fully implemented (**5**)
```
Comment (OlehDulebaEpam) [inline tasks/angular/forms.md:71]: '+' sign is missing in (**5**)

File: tasks/angular/forms.md [hunk 4]
```diff
@@ -41,38 +41,51 @@ The main goal of this task is to implement forms handling and validation using R
...
+- [ ] TODO: seems that form submit logic is missing at any point of the course. To be added if confirmed.
```
Comment (OlehDulebaEpam) [inline tasks/angular/forms.md:72]: Login form and Card creation form (https://github.com/rolling-scopes-school/tasks/blob/master/tasks/angular/NgRX.md) is used in application.
Item have to be removed

Comment (Satlykov) [reply tasks/angular/forms.md:72]: Did it.

File: tasks/angular/forms.md (hunk 3 above)
Comment (Satlykov) [reply tasks/angular/forms.md:71]: Did it.

File: tasks/angular/forms.md (hunk 2 above)
Comment (Satlykov) [reply tasks/angular/forms.md:70]: Did it.

**Pull request merged**